CHANGELOG.md

```diff
@@ -0,0 +1,57 @@
+# Changelog
+
+All notable changes to the Carbon Aware SDK will be documented in this file.
+
+## [Unreleased]
+
+### Added 
+
+- Added Electricity Map support for forecasting.  This supports the commercial Electricity Map API only, and currently does not support the free Electrity Map API due to differences in the API's.
```

Comment:
Also Added Electricity Maps Free (CO2 Signal) support for emissions data (maybe need to reflect this only supports CURRENT emissions data - no Forecast or historical)

Reply:
This needs to be created separately if possible, this changelog is just to create the initial changelog itself.  I can add this one though since it's there now.